recipients_app/lib/view/widgets/welcome/phone_input_page.dart

```diff
@@ -45,6 +48,18 @@ class _PhoneInputPageState extends State<PhoneInputPage> {
         child: Column(
           mainAxisAlignment: MainAxisAlignment.center,
           children: <Widget>[
+            Align(
+              alignment: Alignment.topRight,
+              child: SafeArea(
+                child: ButtonSmall(
+                  onPressed: () {
+                    demoManager.isDemoEnabled = true;
+                  },
+                  label: localizations.demoCta,
+                  buttonType: ButtonSmallType.outlined,
+                ),
+              ),
```

Comment:
Instead of
```
MediaQuery.of(context).size.height
```
use 
```
MediaQuery.sizeOf(context).height
```
for better performance

Reply:
I improved this inn the whole app.